docs/mine-hnt/validators/setup.mdx

```diff
@@ -0,0 +1,161 @@
+
+## Port forwarding
+
+Regardless of using docker or source, before launching a validator node configure the TCP port on your network. Validators communicate to peers over port: 2154/TCP. The networking logic knows how to get around a lack of forwarding, but validators will gain improved performance by forwarding the port.
+
+## <a name="docker-image"></a>From docker image
+
+See technical requirements [here](/mine-hnt/validators/requirements).
+
+First, update the package manager registry:
+```sudo apt-get update```
+
+Then, install Docker:
+
+```sudo apt-get install docker.io```
+
+To avoid needing to use docker with sudo privileges, add your user to the docker group, replacing ```$USER``` with your username:
+
+```sudo usermod -aG docker $USER```
+
+Log in and out of your account to apply these changes. You are now ready to use Docker.
+
+### Run a docker container
+
+Note: on amd64 systems AVX support is required. Verify that it exists on your host system:
+```
+grep avx /proc/cpuinfo
+grep avx2 /proc/cpuinfo
+```
+If nothing is returned from these commands, your host system does not have AVX support and your validator may not be stable.
+
+### Start container
+
+Before running the container for the first time, it is advisable to pick a "system mount point", i.e., a directory in your host system; some long-term data is stored there. This allows you to easily maintain your validator's blockchain identity (i.e. swarm keys) and blockchain state through image updates.
+
+If you are using a Linux system, you can just create a directory in your user's home directory:
+```mkdir ~/miner_data```
+
+If you are using Ubuntu as user ubuntu, this path would now be ```/home/ubuntu/miner_data```. This will be used later.
+Now you can try the ```run``` command to start your container for the first time:
+
+```
+docker run -d \
+--restart always \
+--publish 2154:2154/tcp \
+--name miner \
+--mount type=bind,source=/home/ubuntu/miner_data,target=/var/data \
+quay.io/team-helium/miner:miner-xxxNN_YYYY.MM.DD
+```
+
+Replace ```xxxNN``` with the architecture used, i.e. amd64 or arm64, and with the release date of the image.
+
+The ```-d``` option runs in detached mode, which makes the command return or not; you may want to omit if you have a daemon manager running the docker for you.
+
+The ```--restart always``` option asks Docker to keep the image running, starting the image on boot and restarting the image if it crashes. Depending on how you installed Docker in your system, it'll start on boot. In the AWS AMI above, we use systemd (```systemctl status docker``` to check).
+
+The ```--name miner``` names the container, which makes interacting with the docker easier, but feel free to name the container whatever you want.
+
+The ```--mount``` with the parameters above will mount the container's ```/var/data/``` directory to the systems directory ```/home/ec2-user/miner_data```.
```

Comment:
```suggestion
The ```--mount``` with the parameters above will mount the container's ```/var/data/``` directory to the systems directory ```/home/ec2-user/validator_data```. You will want this folder to be persistent across runs of the docker container as it will contain both the blockchain data and the miner key of your validator. 
```